Source/AliveLibAO/Paramite.cpp

```diff
@@ -1059,7 +1052,7 @@ void Paramite::State_0_Idle_44B900()
         }
         else
         {
-            MusicController::sub_443810(MusicController::MusicTypes::eType12, this, 0, 0);
+            MusicController::sub_443810(MusicController::MusicTypes::eType11, this, 0, 0);
```

Comment:
this used to be 12?

Reply:
this was previously 12 but should've been 11